docs/recipes/recipes.md

```diff
@@ -889,6 +889,81 @@ Example := Window {
 }
 ```
 
+### Breakpoints for responsive ui
+
+Uses this recipe to implement a responsive SideBar that collapses when the parent width is smaller then the given break-point. On Button
+click the SideBar can be expanded if collapsed on break-point.
+
+```slint,no-auto-preview
+import { Button } from "std-widgets.slint";
+
+export SideBar := Rectangle {
+    private property <bool> collapsed: reference-width < break-point;
+    property <length> reference-width;
+    property <length> break-point: 600px;
+
+    width: 160px;
+   
+    container := Rectangle {   
+        private property <bool> expaned;
+        width: parent.width;
+        background: black;
+
+        VerticalLayout { 
+            padding: 4px;
+            alignment: start;
+            if (collapsed) : Button {
+                checked: container.expaned;
+                text: "C";
+
+                clicked => {  
+                    container.expaned = !container.expaned;
+                }
```

Comment:
I tried in the online editor, and the "C" is padded a bit too far, I think:

<img width="156" alt="Screenshot 2022-12-16 at 12 14 25" src="https://user-images.githubusercontent.com/1486/208086566-13c15def-7ef7-47ff-aed6-30709c9ee34e.png">